site/src/pages/loci/[id].astro

```diff
@@ -70,6 +77,15 @@ const hasRawSearch = Object.keys(
         <div><TbDna2 />Gene ID</div>
         <div>{locus.gene}</div>
       </div>
+
+      <div class="box">
+        <div><FaRegCalendar />Updated</div>
+        <div>
+          {blame.date}
+          <br />
+          {blame.version}
```

Comment:
Suggest puting "v" or "STRchive v" in front of this.

e.g.
Updated
Dec 9, 2024
STRchive v2.0.0

or 

Updated
Dec 9, 2024
v2.0.0

Reply:
Feel free to make tweaks to this as you see fit. I was also thinking of not putting this info in one of the gray boxes, and instead putting it as a super script to the right of the top heading (the id), or something like that. That'd require some CSS though.